src/contrib/mlir/dialect/ops.td

```diff
@@ -260,5 +260,46 @@ def NGAnyRedOp : NG_Axis_Reduction_Op<"any.red">
 // Terminator Ops
 def NGReturnOp : NG_Terminator_Op<"return">;
 
+// Quantized Ops
+def NGQuantizeOp :
+      NG_OneResult_Op<"quantize", [NoSideEffect]>,
+      Arguments<(ins NG_TensorType:$input, NG_TensorType:$scale, NG_TensorType:$zeroPoint,
+                 I64Attr:$outputType, I64ArrayAttr:$axes, I64Attr:$roundMode)>
```

Comment:
`outputType` should be `TypeAttr` instead. We should also add Quantized Types as well.

Reply:
Same for Dequantize.